packages/picasso/src/PageTopBar/PageTopBar.tsx

```diff
@@ -124,7 +124,10 @@ export const PageTopBar = forwardRef<HTMLElement, Props>(function PageTopBar(
           {/*  Left part: Hamburger, Logo, Tagline, Search bar */}
           <div className={classes.left}>
             <Container flex alignItems='center' gap='small'>
-              <PageHamburger id={hamburgerId} />
+              <PageHamburger
+                id={hamburgerId}
+                data-testid='page-hamburger-button'
```

Comment:
```suggestion
                data-testid={testIds.hamburger}
```
We should not have hardcoded test ids in Picasso, we should define them in tests

Reply:
@TomasSlama Thank you, fixed! Now it is coming from TopBar and is not hardcoded

Reply:
@sashuk 👍 I think you need to update snapshots

Reply:
@TomasSlama Yep, about to push them!